commons/servers/QueryParameters.ts

```diff
@@ -12,10 +12,11 @@ function convertFiltersToQueryParams(filters?: Record<string, any>): Map<string,
   const entries = Object.entries(filters)
     .filter(([_, value]) => !!value)
     .map<[string, string[]]>(([name, value]) => {
-      const newName = name.endsWith('s') ? name.slice(0, -1) : name
+      let newName = name
```

Comment:
Before this change, the standard was that all variables that ended with `s` had the `s` removed. This was because we wanted to convert `entityTypes: [ type1, type2 ]` to `&entityType=type1&entityType=type2`.

But `fields` is as string, so are now only making that "s modification" when the value is an array